### DISCUSSION TITLE
Universe 2024 Wishlist: What do hope to see announced?

### DISCUSSION THREAD

**Discussion by novium2** [community/community#142661] [Other Feature Feedback, Questions, & Ideas]
Tell us all the things that would make this the best Universe 2024. What updates would you want to see announced? What new features, new products, improvements?

**Comment by tonnytg** [comment]
Hey @novium2, I would love to see more customization options in GitHub's features section for configuring GitHub Copilot in 2024.

For example, the ability to filter suggestions specifically for comments, variable names, or certain types of code blocks like functions or object-oriented structures, helps to preserve the unique characteristics of the source code.

Additionally, it would be helpful if GitHub provided a usage percentage indicator for suggested patterns across repositories. For instance, "This code block suggestion has been used by 35% of repositories."

**Reply by novium2** [reply]
That's a great idea, I especially like the percentage one.
